tests/Unit/RemoteOutputSizeLimitsTest.php

```
@@ -0,0 +1,124 @@
+        ->and(strlen($taskOutput))->toBeGreaterThan(ScheduledTaskJob::MAX_OUTPUT_SIZE_BYTES)
+        ->and($taskOutput)->toEndWith('[... Output truncated at 5MB limit ...]');
+});
+
+it('does not pass the scheduled task output wrapper through the sudo rewriter', function () {
+    $source = remoteOutputSource('app/Jobs/ScheduledTaskJob.php');
+    $reflection = new ReflectionClass(ScheduledTaskJob::class);
+    $method = $reflection->getMethod('boundedTaskCommand');
+    $command = $method->invoke($reflection->newInstanceWithoutConstructor(), 'sudo docker exec example true');
+    $server = Mockery::mock(Server::class)->makePartial();
+    $server->shouldReceive('getAttribute')->with('user')->andReturn('ubuntu');
+    $rewrittenCommand = parseCommandsByLineForSudo(collect([$command]), $server)[0];
```

PHPStan (2.2.7)

[warning] Parameter `#2` $server of function parseCommandsByLineForSudo expects App\Models\Server, Mockery\LegacyMockInterface given.


(argument.type)